src/Graph/LookupTable.cpp

```diff
@@ -34,4 +39,63 @@ namespace Graph {
     SKEE::INiTransformInterface* LookupTable::GetNiTransformInterface() {
         return niTransformInterface;
     }
+
+    void parseActor(json& json, ActionActor& actor) {
+        if (json.contains("stimulation")) {
+            actor.stimulation = json["stimulation"];
+        }
+        else {
+            actor.stimulation = 0.0;
```

Comment:
floats cannot be null, so this will be initialized to 0.0 anyways.

Reply:
I thought it would init to whatever value was in the memory at the time, so could be any random value